src/main/scala/beam/utils/scenario/urbansim/censusblock/UrbansimReaderV2.scala

```diff
@@ -21,15 +21,15 @@ class UrbansimReaderV2(
   private val logger = LoggerFactory.getLogger(getClass)
 
   private val inputHouseHoldMap: Map[String, InputHousehold] = {
-    logger.debug("Start reading of households info...")
+    logger.info("Start reading of households info...")
```

Comment:
Do you want this for the final merge - or was it only for easier debugging?

Reply:
I changed it in order to have better visibility because of some faced issue. Would prefer to leave it on info level in case something will go wrong.